tests/arrays.rs

```diff
@@ -109,31 +109,24 @@ fn iter() {
 
 #[test]
 fn empty() {
-    assert!(<[u32; 0]>::iter_paths::<2, String>("")
-        .unwrap()
-        .next()
-        .is_none());
+    assert!(<[u32; 0]>::iter_paths::<String>("").next().is_none());
 
     #[derive(Miniconf, Serialize, Deserialize)]
     struct S {}
 
-    assert!(miniconf::Array::<S, 0>::iter_paths::<2, String>("")
-        .unwrap()
+    assert!(<[S; 0] as Miniconf>::iter_paths::<String>("")
+        .next()
+        .is_none());
+    assert!(<[[S; 0]; 0] as Miniconf>::iter_paths::<String>("")
         .next()
         .is_none());
-    assert!(
-        miniconf::Array::<miniconf::Array<S, 0>, 0>::iter_paths::<2, String>("")
-            .unwrap()
-            .next()
-            .is_none()
-    );
 
     #[derive(Miniconf)]
     struct Q {
         #[miniconf(defer)]
```

Comment:
```suggestion
        #[miniconf(defer(2))]
```

I don't think the change to the test is equivalent, not sure if 2 is the correct value.

Reply:
Right!